### ISSUE TITLE
Extra LDAPHOST env var for LDAP service

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/docker#223]
What is it for @jeanmi151 ?
https://github.com/georchestra/docker/blob/b4b553a98bdd9a442001b995352a282d0c1982c5/docker-compose.yml#L74

**Comment by jeanmi151** [comment]
something forgotten  ...
will remove it

**Comment by jeanmi151** [comment]
huuuum aaa no, it is for saying that for the container ldap, ldaphost is the localhost, otherwise it will try to contact it (and it will not work), If I remember correctly , will test it

**Comment by jeanmi151** [comment]
Yes, the container fail to start if LDAPHOST if not set at "localhost", I will add a comment

**Comment by jeanmi151** [comment]
https://github.com/georchestra/docker/commit/9aeb4babbd3302e31d6bd4b04de34e448d9741a1 , will cherry-pik on master too

**Comment by jeanmi151** [comment]
https://github.com/georchestra/docker/pull/224
